coalib/bears/requirements/PythonRequirement.py

```diff
@@ -5,6 +5,37 @@ class PythonRequirement(PackageRequirement):
     """
     This class is a subclass of ``PackageRequirement``, and helps specifying
     requirements from ``pip``, without using the manager name.
+
```

Comment:
why is it confusing? it explains all cases in which you would use the method and how it would work, imo its useful?